lmctl/client/api/assemblies.py

```diff
@@ -81,10 +93,13 @@ def _intent_request_impl(self, intent_name: str, intent_obj: Union[Dict,
                                                                         HealAssemblyIntent, 
                                                                         ScaleAssemblyIntent,
```

Comment:
Need to add `RetryAssemblyIntent` and the other intent types to the list of accepted types

Reply:
Addressed